### PR TITLE
Hex

### PR DESCRIPTION
**Pull request by davidmanzanares**

### CODE REVIEW

File: src/core/style/expressions/hex.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+/**
+ *
+ * Create a color from their hexadecimal description
+ *
+ * @param {string} hexadecimalColor - color in the form #AA88AA
+ * @return {carto.style.expressions.hex}
+ *
+ * @example <caption>Display blue points.</caption>
+ * const s = carto.style.expressions;
+ * const $type = s.property('type');
```
Comment (Jesus89) [inline src/core/style/expressions/hex.js]: $type not used

File: src/core/style/expressions/hex.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import Expression from './expression';
+import { checkString, hexToRgb, getStringErrorPreface } from './utils';
+
+/**
+ *
+ * Create a color from their hexadecimal description
```
Comment (IagoLast) [inline src/core/style/expressions/hex.js]: Create a color from `its` hexadecimal description?

File: src/core/style/expressions/utils.js [hunk 3]
```diff
@@ -15,12 +15,18 @@ export function implicitCast(value) {
 }
 
 export function hexToRgb(hex) {
+    // TODO allow for #F00 and #F004 notations
```
Comment (IagoLast) [inline src/core/style/expressions/utils.js]: Maybe we should point in the doc this forms are not supported

**Comment by Jesus89** [comment]
Added support for `#ABC` colors

**Pull request merged**